cmd/harmony/flags.go

```diff
@@ -271,6 +278,10 @@ func applyGeneralFlags(cmd *cobra.Command, config *harmonyConfig) {
 	} else if cli.IsFlagChanged(cmd, legacyDataDirFlag) {
 		config.General.DataDir = cli.GetStringFlagValue(cmd, legacyDataDirFlag)
 	}
+
+	if cli.IsFlagChanged(cmd, isOfflineFlag) {
+		config.General.IsOffline = cli.GetBoolFlagValue(cmd, isOfflineFlag)
```

Comment:
Shall we add here?

```
isOffline : =cli.GetBoolFlagValue(cmd, isOfflineFlag)
config.General.IsOffline = isOffline
if isOffline {
     config.P2P.IP = nodeconfig. DefaultLocalListenIP
}
```

Reply:
Hmm, well the current impl already accounts for this. 
If we want to implicitly set the IP, we should prob do it in `applyP2PFlags`.

Reply:
Prob makes sense to implicitly set the IP, ill update.